decidim-core/lib/decidim/core.rb

```diff
@@ -5,6 +5,7 @@
 require "decidim/core/version"
 # Decidim configuration.
 module Decidim
+  autoload :DummyTranslator, "decidim/dummy_translator"
```

Comment:
Do we need this translator in production? Shouldn't it be loaded by `decidim-dev` or configured in an initializer, the development_app for example?

Reply:
I think we found some problems with not having it here, but yes, maybe this autoload can be moved to `decidim-dev/lib/decidim/dev.rb`?

Reply:
Moving this autoload to `decidim-dev` implies moving the whole `DummyTranslator` class there, at least. I'll explore that.

Reply:
Should be done now 😄